### ISSUE TITLE
Add support for multiple reqPathSearch/reqPathReplace expressions

### ISSUE THREAD

**Issue by cazacugmihai** [vfarcic/docker-flow-proxy#383]
Please add support for multiple expressions, like:

```
--label com.df.reqPathSearch='/image' \
--label com.df.reqPathReplace='/media/image' \
--label com.df.reqPathSearch='/webpack-dev' \
--label com.df.reqPathReplace='/sockjs-node' \
```

Thanks!

**Comment by cazacugmihai** [comment]
As for the syntax, we can use something like this:

```
--label com.df.reqPathReplace=/image:/media/image \
--label com.df.reqPathReplace=/webpack-dev:/sockjs-node
```
where `com.df.reqPathReplace` allows a `key:value` value where `:` is the delimiter.

In order to make this delimiter customizable, a new label can be created:
```
--label com.df.reqPathReplace.delimiter=:
```

**Comment by vfarcic** [comment]
Sounds good but I don't think I'll have time before the next weekend. I hope that's OK.

**Comment by cazacugmihai** [comment]
Yes, there is no rush. Thank you!

**Comment by vfarcic** [comment]
I promised to work on this issue this weekend but I could not manage to get enough time. There were too many open issues and other tasks I had to work on. Next weekend...

**Comment by vfarcic** [comment]
It's available from the [release 17.12.09-73](https://github.com/vfarcic/docker-flow-proxy/releases/tag/17.12.09-73).

**Comment by cazacugmihai** [comment]
Thanks Viktor for this improvement!
